src/components/NewMemberSection/NewMemberCard/index.tsx

```diff
@@ -8,6 +8,7 @@ import {
 import { RootState } from "@/src/store";
 import { useGetIsSuperUser } from "@/src/utils/customHooks";
 import { useRouter } from "next/router";
+import { UserType } from "../../MembersSectionNew/types/MembersSection.type";
```

Comment:
use absolute import if possible

Reply:
[i have same answer as above](https://github.com/Real-Dev-Squad/members-site/pull/118#discussion_r1407632774)

Reply:
This one is components i guess there is no link to jest

Reply:
Test for this component is already written but when kotesh was working on it he just added type and didn’t import it so i fixed that